matcher/service/openreview_interface.py

```diff
@@ -116,6 +89,36 @@ def __init__(self, client, config_note_id, logger=logging.getLogger(__name__), c
                 self.set_status('Error')
                 raise error_handle
 
+    def get_all_edges(self, client, edge_invitation_id):
+        '''Helper function for retrieving and parsing all edges in bulk'''
+
+        all_edges = []
+        all_papers = {p:i for i,p in enumerate(self.papers)}
+        all_reviewers = {r:i for i,r in enumerate(self.reviewers)}
+        self.logger.debug('GET invitation id={}'.format(edge_invitation_id))
+        edge_invitation = client.get_invitation(edge_invitation_id)
```

Comment:
is there a reason to get the edge invitation? I'm not sure if we need this information.

Reply:
Well it was being used by build_edge which I am not calling anymore in this function. I will remove it now.